OpenTranslator/Controllers/Awesome/AdminController.cs

```diff
@@ -353,7 +353,34 @@ public bool doesTextIdExist(AdminInput input)
 				return false;
 			}
 		}
+		[HttpPost]
```

Comment:
check the previous comment, this method would be removed or replace by the one I suggest in previous comment (use pascal case)

Reply:
Yes, I have removed my function and used form Translation Repository.